src/components/resource-selector/shared/resource-preview-screen.yml

```diff
@@ -2,3 +2,6 @@ resourcePreviewScreen:
   headline:
     en: File preview
     de: Datei-Vorschau
+  missing:
+    en: missing
+    de: fehtl
```

Comment:
Should be "fehlt". But unfortunately also that doesn't work together with both the singular label "Beschreibung" and plural label "Sprachen" (because there it should be "fehlen"). Alternatives:

* "not provided/specified"/"nicht angegeben"
* "none"/"keine (Beschreibung/Sprachen)" (although it might have a language that was just not provided)

To me, the first bullet seems to fit best, because "missing" sounds rather deficient, as if something were wrong with file.

Reply:
Agreed (also, I noticed that "fehlt" does not work for plural but didn't know what else to use). Changed, thanks!